### DISCUSSION TITLE
Template submission

### DISCUSSION THREAD

**Discussion by scosman** [vercel/community#5505] [Help] [answered]
### Summary

I can't seem to find an official form for template submissions for https://vercel.com/templates

Can I request it here (if not, please point me in the right direction)?

Projects is SaaS template/boilerplate built with SvelteKit and hostable on Vercel. It’s called [SaasStarter.work](https://SaasStarter.work)

Details:

- MIT Open Source: https://github.com/CriticalMoments/CMSaasStarter
- Feature Rich: user auth, user dashboard, marketing site, billing/subscriptions, pricing page, blog engine, and more
- Very fast: perfect PageSpeed scores, achieved with pre-rendering and pre-compiling.
- Extensible: all the tools you need to make additional marketing pages, UI components, admin portals, database backends, API endpoints, etc.
- Hosting: easy to manage, automatic deployments

### Steps to Reproduce

NA

**Comment by amyegan** [comment, marked as answer]
Hey @scosman. We just re-opened Templates Marketplace for new submissions. If you're still interested in sharing your template, please share it with us using the [Templates Marketplace submission form](https://vercel.com/templates/submit) 🙂
